src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectConfigurationFilePathStore.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem
+{
+    internal abstract class ProjectConfigurationFilePathStore
+    {
+        public abstract event EventHandler<ProjectConfigurationFilePathChangedEventArgs> Changed;
+
+        public abstract IReadOnlyDictionary<string, string> GetMappings();
+
+        public abstract void Set(string projectFilePath, string configurationFilePath);
+
+        public abstract bool TryGet(string projectFilePath, out string publishFilePath);
```

Comment:
```suggestion
        public abstract bool TryGet(string projectFilePath, out string configurationFilePath);
```
Using different terminology is confusing.

Reply:
Also, I see a lot of the places in the code refer to this as publishPath. I don't have a preference as long as we use the same name (Config or Publish) everywhere.

Reply:
ahhh, looks like I missed some renames is all 😄